PERA-1545 :: Compose migration of BaseInfoFragment

alexs60 opened the pull request:
# Pull Request Template

## Description
- Converting BaseInfoFragment and all child Fragment to Compose
- Created compose Theme ( still need to be updated with the Figma system design)
- Created Font, Text components and Buttons for compose uis

https://github.com/user-attachments/assets/eb17f14f-aacd-406b-8321-745493924ba1


Here is is a sample of how it looks like before and after. (right is XML, left is compose)

**Light**

https://github.com/user-attachments/assets/4136c333-73c8-4e4d-9716-62dc4c7e97e7

**Dark**

https://github.com/user-attachments/assets/e8c7c1fd-00a8-4e13-af7e-c541a2ee853f




## Related Issues
- Closes https://algorandfoundation.atlassian.net/browse/PERA-1545

## Checklist
- [x] Have you tested your changes locally?
- [x] Have you reviewed the code for any potential issues?
- [ ] Have you documented any necessary changes in the project's documentation?
- [ ] Have you added any necessary tests for your changes?
- [ ] Have you updated any relevant dependencies?

## Additional Notes
- Add any additional notes or comments that may be helpful for reviewers.

alexs60 committed:
migration of BaseInfoFragment to Compose

alexs60 committed:
migration of BaseInfoFragment to Compose

alexs60 committed:
applying theming for Compose

alexs60 committed:
applying theming for Compose

alexs60 committed:
deleting wrong db files

alexs60 committed:
deleting wrong db files

michaeltchuang commented:
i noticed this when you click recover account screen

![Screenshot_1738962472](https://github.com/user-attachments/assets/fec344ad-670f-456a-943c-788d7e4c99f3)

michaeltchuang commented:
Another screen...i think second button text color is off (this screen is when click create account)

![Screenshot_1738963549](https://github.com/user-attachments/assets/feb316fc-593e-46bb-8c45-15fcda7c7dbe)

michaeltchuang reviewed (changes requested):
let's fix the colors issue before merging

michaeltchuang committed:
remove unused fragment_base_info.xml since it's in compose now

alexs60 committed:
Merge branch 'dev' into feature/compose-migration

alexs60 committed:
Merge branch 'feature/compose-migration' of github.com:alexs60/pera-android-fork into feature/compose-migration

alexs60 committed:
aligned styles in the compose buttons and sizes of text

alexs60 commented:
@michaeltchuang Hi, I've checked and updated the colors accordingly also the text size was actually too little. fixed it as well.
![screen-1](https://github.com/user-attachments/assets/40eeb1fb-a84c-47b1-97d0-f0f64d459fb2)

alexs60 committed:
warning text alignment to xml style

alexs60 committed:
adjust sizing based on xml ui

alexs60 committed:
PeraIconBig and Dark theme fixes

michaeltchuang committed:
update browserstack github action to run on forked repos

michaeltchuang reviewed (approved):
screens and colors look acceptable enough to merge for me
